server/src/test/java/org/elasticsearch/action/bulk/TransportBulkActionIngestTests.java

```diff
@@ -460,7 +462,7 @@ public void testUseDefaultPipelineWithBulkUpsert() throws Exception {
         verifyZeroInteractions(transportService);
     }
 
-    public void testCreateIndexBeforeRunPipeline() throws Exception {
+    public void testDoExecuteCalledTwiceCorrectly() throws Exception {
```

Comment:
This test is a good test, but it's name was slightly misleading (it still passed with my change to run pipeline before the create index)